### ISSUE TITLE
When fielsets has only one fieldset, carousel, horizontal_tabs, verical_tabs will not be displayed

### ISSUE THREAD

**Issue by fengyouchao** [farridav/django-jazzmin#103]
When `fielsets` has only one `fieldset`, `carousel`, `horizontal_tabs`, `verical_tabs` will not be displayed

For example:

```
fieldsets = (
    ('Tab1', {
        'fields': (
            'id',
        ),
    }),
)
```

Tab1 will not display.

## Override get_fieldsets

This also affects `get_fieldsets`. Only when `fieldsets` are set and contain more than one `fieldset`, the `fieldsets` returned by the overridden `get_fieldsets` will be displayed correctly.

Case 1

```
# fieldsets is not set
def get_fieldsets(self, request, obj=None):
    return (
        (_('Tab3'), {
            'fields': (
                'id',
            ),
        }),
    )
```

Tab3 will not display.

Case 2:

```
fieldsets = (
    ('Tab1', {
        'fields': (
            'id',
        ),
    }),
)

def get_fieldsets(self, request, obj=None):
    return (
        (_('Tab3'), {
            'fields': (
                'id',
            ),
        }),
    )
```

Tab3 will not display.

Case 3

```
fieldsets = (
    ('Tab1', {
        'fields': (
            'id',
        ),
    }),
    (_('Tab2'), {
        'fields': (
            'release_channel',
        ),
    }),
)

def get_fieldsets(self, request, obj=None):
    return (
        (_('Tab3'), {
            'fields': (
                'id',
            ),
        }),
    )
```

Tab3 will be displayed correctly.

In other words, if `get_fieldsets` is overridden, in order to be displayed correctly, `fieldsets` must be set, and there must be at least two `fieldset`.
